app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/returns/SingleDisposalTriageAnswers.scala

```diff
@@ -122,6 +124,15 @@ object SingleDisposalTriageAnswers {
         case _                           => None
       }
 
+    def isPeriodOfAdmin(): Boolean =
+      s.fold[Option[IndividualUserType]](
```

Comment:
this can be shortened to just `representativeType.contains(PersonalRepresentativeInPeriodOfAdmin)`